src/__tests__/Registration.test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducers from '../redux/reducers/index';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+// import  RegistrationPage  from '../components/pages/Registration/RegisterStep1';
+import InstructorWelcome from '../components/pages/Registration/InstructorWelcome';
+
+const store = createStore(rootReducers);
+
+jest.mock('@okta/okta-react', () => ({
+  useOktaAuth: () => {
+    return {
+      authState: {
+        isAuthenticated: true,
+      },
+      authService: {},
+    };
+  },
+}));
+
+describe('<Instructor Welcome />', () => {
+  test('it renders the instructorWelcome component', () => {
+    const { getByText } = render(
```

Comment:
the tests look simple and seem to work great, why does getBytext have this linter warning?